React/Views/RCTWebViewManager.m

```diff
@@ -38,7 +38,6 @@ - (UIView *)view
 RCT_REMAP_VIEW_PROPERTY(scrollEnabled, _webView.scrollView.scrollEnabled, BOOL)
 RCT_REMAP_VIEW_PROPERTY(decelerationRate, _webView.scrollView.decelerationRate, CGFloat)
 RCT_EXPORT_VIEW_PROPERTY(scalesPageToFit, BOOL)
-RCT_EXPORT_VIEW_PROPERTY(messagingEnabled, BOOL)
```

Comment:
Was this intentional? We use this to know whether we should override `window.postmessage`.